content/applications/hr/payroll.rst

```diff
@@ -415,39 +649,57 @@ field.
+
+To make a new type of salary attachment, click the :guilabel:`New` button. Then, enter the
+:guilabel:`Name`, the :guilabel:`Code` (used in the salary rules to compute payslips), and decide if
+it should have :guilabel:`No End Date`.
+
+.. image:: payroll/new-attachment.png
+   :align: center
+   :alt: A new salary attachment form filled in.
 
 Salary package configurator
 ===========================
 
 The various options under the :guilabel:`Salary Package Configurator` section of the
-:menuselection:`Payroll --> Configuration --> Salary Package Configurator` menu all affect an
-employee's potential salary. These sections (:guilabel:`Advantages`, :guilabel:`Personal Info`,
-:guilabel:`Resume`, and :guilabel:`Offers`) specify what benefits can be offered to an employee in
-their salary package.
+:menuselection:`Payroll app --> Configuration --> Salary Package Configurator` menu all affect an
+employee's potential salary. These sections (:guilabel:`Benefits`, :guilabel:`Personal Info`,
+and :guilabel:`Resume`) specify what benefits can be offered to an employee in their salary package.
 
 Depending on what information an employee enters (such as deductions, dependents, etc.), their
 salary is adjusted accordingly. When an applicant applies for a job on the company website, the
 sections under :guilabel:`Salary Package Configurator` directly affect what the applicant sees, and
-what is populated as the applicant enters information.
+what is populated, as the applicant enters information.
 
-Advantages
-----------
+Benefits
+--------
 
-When offering potential employees a position, there can be certain *advantages* or benefits set in
-Odoo in addition to the salary to make an offer more appealing (such as extra time off, the use of a
-company car, reimbursement for a phone or internet, etc.).
+When offering potential employees a position, there can be certain benefits set in Odoo, in addition
+to the salary, to make an offer more appealing (such as extra time off, the use of a company car,
+reimbursement for a phone or internet, etc.).
 
-To see the advantages, go to :menuselection:`Payroll --> Configuration --> Salary Package
-Configurator --> Advantages`. Advantages are grouped by :guilabel:`Structure type`, and the
-advantage listed for a particular structure type is only available for that specific structure.
+To view the benefits, go to :menuselection:`Payroll app --> Configuration --> Salary Package
+Configurator: Benefits`. Benefits are grouped by :guilabel:`Structure type`, and the benefit listed
+for a particular structure type is only available for that specific structure.
 
-.. image:: payroll/advantages.png
+.. image:: payroll/benefits.png
    :align: center
-   :alt: A list view of all the advantages or benefits that is available for each structure type.
+   :alt: A list view of all the benefits available for each structure type.
 
 .. example::
```

Comment:
this example seems repetitive. please ensure it only conveys the example being described once, please. Thanks!

Reply:
I understand what you mean, but I wanted to walk them through the example. So first, I explain what is where, and then I explain who gets what.  I shortened it a bit so it was less verbose